tests/datalab/test_datalab.py

```diff
@@ -1113,3 +1113,23 @@ def test_find_imbalance_issues_no_args(self, imbalance_labels):
         assert "class_imbalance" in summary["issue_type"].values
         class_imbalance_summary = lab.get_issue_summary("class_imbalance")
         assert class_imbalance_summary["num_issues"].values[0] > 0
+
+class TestSpuriousCorrelations: 
+
+    def test_spurious_correlations(self): 
+        dark_scores = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
+        labels = np.array([0, 1, 0, 1, 0, 1, 0, 1, 0, 1])
+        issue_summary = pd.DataFrame({
+            'issue_type': ['dark'],
+            'num_issues': 10,
+        })
+
+        data = {
+            'issues': pd.DataFrame({'dark_score': dark_scores}),
+            'labels': labels,
+            'issue_summary': issue_summary,
+        }
```

Comment:
this not how the test should be. The test should be:

1) construct a toy image dataset (say with 10-20 images)
2) run `datalab.find_issues(..., image_key = ...)` on this toy dataset, following the tutorial here:
https://docs.cleanlab.ai/master/tutorials/image.html

3) run `corrs = datalab._spurious_correlations()` after that.

4) verify the results in `corrs` are as expected via `assert` statements.

Reply:
you can see easy way to construct a toy image here:
https://github.com/cleanlab/cleanvision/blob/main/tests/test_image_property_helpers.py

using: 

```
from PIL import Image
img = Image.new("RGB", (200, 200), (255, 0, 0))
```

Reply:
hello @jwmueller can you please review this and help me to write condition for assert statement
```
class TestSpuriousCorrelation:
    def create_data(self):
        images = [Image.new("RGB", (32, 32), (25, 25, 25))] * 5 + \
                 [Image.new("RGB", (32, 32), (255, 255, 255))] * 5

        rand_images = (np.random.rand(40, 32, 32, 3) * 255).astype(np.uint8)
        images = images + [Image.fromarray(img) for img in rand_images]

        labels = np.array([0] * 5 + [1] * 5 + [2] * 40)
        data = {
            "image": images,
            "label": labels
        }
        dataset = Dataset.from_dict(data)
        lab = Dataset(data=dataset, image_key="image", label_name="label")
        features = np.array([np.array(img).flatten() for img in images])
        return lab, features

    def test_spurious_correlation(self):
        imagelab, features = self.create_data()
        imagelab.find_issues(features=features)
        corrs = imagelab._spurious_correlation()
        assert corrs
```

Reply:
which property of the image here is the one correlated with the labels?  Hard for me to tell just from your code.  

If it is say the `dark` score property, then your assert should verify that corrs[dark] > corrs[i] for all other image properties i.

Reply:
i wrote complete code to address this <https://colab.research.google.com/drive/1Pvu7QES5AgUQjTClwJAiJeHkKlhqW4sW#scrollTo=t9XAO03pRnfH&uniqifier=1>